tests/integration/targets/ec2_asg_instance_refreshes_info/tasks/refresh_and_cancel_three_times.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: test starting a refresh with an ASG name
+  ec2_asg_instance_refresh:
+    name: "{{ resource_prefix }}-asg"
+    state: "started"
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    region: "{{ aws_region }}"
+  ignore_errors: no
+  retries: 10
+  delay: 5
+  register: refreshout
+  until: refreshout is not failed
+- debug: var=refreshout
+
+- name: test canceling a refresh with an ASG name
+  ec2_asg_instance_refresh:
+    name: "{{ resource_prefix }}-asg"
+    state: "canceled"
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    region: "{{ aws_region }}"
```

Comment:
Shouldn't be needed and won't work in CI because we use session tokens.

Reply:
Fixed in 958a04ab7f544eb8711a22f741283366d5f233d3